### PR TITLE
feat(client) MaxContentLength configurable field

### PR DESCRIPTION
**Pull request by andrewgkew**
Created a new cli.conf.yaml field to set the MaxContentLength value for the REST client used when upload or reading files using portal CLI tool

### CODE REVIEW

**Comment by zekth** [comment]
i feel like `WorkspaceConfig` and `workspaceConfig` are confusing. Can we extract the static from `WorkspaceConfig` in a `constant.ts` file or something?

File: src/core/HTTP/RestClient.ts [hunk 1]
```diff
@@ -62,8 +62,11 @@ export default class RestClient {
     this.client = axios.create({
       baseURL: this.clientUrl,
       headers: this.clientHeaders,
-      httpAgent: new HTTPAgent({ keepAlive: true }),
-      maxContentLength: 10 * 1000000, // 10mb is kong file system
+      httpAgent: new HTTPAgent({ keepAlive: true, maxSockets: 10 }),
```
Comment (zekth) [inline src/core/HTTP/RestClient.ts:65]: why adding the `maxSockets`?

Comment (andrewgkew) [reply src/core/HTTP/RestClient.ts:65]: Ah that was during my testing, will remove!

**Comment by andrewgkew** [comment]
@zekth changes requested have been made:
1. Removed maxSockets
2. Moved constants into separate file

**Comment by nateslo** [comment]
> @zekth changes requested have been made:
> 
> 1. Removed maxSockets
> 2. Moved constants into separate file

I'm still seeing the `maxSockets` in the diff

File: src/core/HTTP/RestClient.ts [hunk 2]
```diff
@@ -62,8 +63,9 @@ export default class RestClient {
...
```
Comment (nateslo) [inline src/core/HTTP/RestClient.ts:66]:
```suggestion
      httpAgent: new HTTPAgent({ keepAlive: true }),
```

Comment (andrewgkew) [reply src/core/HTTP/RestClient.ts]: Yup apologies seemed to have missed that. I have now removed it in my latest commit

**Comment by andrewgkew** [comment]
@nateslo is there anything else you need for this PR to be merged?

File: README.md [hunk 3]
```diff
@@ -70,6 +70,7 @@ Where `<command>` is one of:
 In addition to `kong_admin_uri` and `kong_admin_token`, you can also add:
 - `disable_ssl_verification: true` and
 - `ignore_specs: true`
+- `maxContentLengthInMb` - maximum size allowed to be transferred in Mb - default is 10 (10Mb)
```
Comment (Mierenga) [inline README.md:73]: Can we use snake case like the other options?
```suggestion
- `max_content_length_mb` - maximum size allowed to be transferred in Mb - default is 10 (10Mb)
```

**Review by Mierenga — APPROVED** [review]
LGTM

**Pull request merged**